package.json

```diff
@@ -5,20 +5,27 @@
   "main": "dist/server.bundle.js",
   "scripts": {
     "bootstrap": "shx rm -rf dist && shx mkdir dist",
-    "start": "npm run build && npm-run-all --parallel server watch:**",
-    "build": "npm run build:config && npm run build:client && npm run build:server",
-    "build:server": "webpack --config webpack.server.config.js",
-    "build:client": "npm run bootstrap && npm run build:assets && npm run build:js && npm run build:scss && npm run build:static",
-    "build:assets": "cp -r assets dist",
+    "start": "npm run build && run-p server watch:**",
+    "build": "run-s bootstrap build:**",
     "build:config": "node js/build-config.js > config/env.generated.json",
-    "build:js": "webpack --optimize-minimize --display-error-details",
-    "build:scss": "node-sass scss/main.scss dist/css/main.css && postcss --use autoprefixer --autoprefixer.browsers 'last 3 versions' -o dist/css/main.css dist/css/main.css",
-    "build:static": "cp -r node_modules/mofo-bootstrap/dest/css dist && cp -r node_modules/font-awesome/css/font-awesome.min.css dist/css && cp -r node_modules/font-awesome/fonts dist && postcss --use autoprefixer --autoprefixer.browsers 'last 3 versions' -o dist/css/mofo-bootstrap.css dist/css/mofo-bootstrap.css && cp index.html dist/index.html && cp favicon.png dist && cp manifest.json dist",
+    "build:client": "run-s build-client:**",
+    "build-client:js": "webpack --config webpack.config.js --optimize-minimize --display-error-details",
+    "build-client:scss": "node-sass scss/main.scss dist/css/main.css && npm run build-client:prefix:main",
+    "build-client:prefix:main": "postcss --use autoprefixer --autoprefixer.browsers 'last 3 versions' -o dist/css/main.css dist/css/main.css",
+    "build-client:static": "run-s copy:**",
+    "build:server": "webpack --config webpack.server.config.js --optimize-minimize --display-error-details",
+    "copy:styling-dependency": "run-s bootstrap:clean bootstrap:fontawesome bootstrap:prefix",
+    "bootstrap:clean": "shx cp -r node_modules/mofo-bootstrap/dest/css dist",
+    "bootstrap:fontawesome": "shx cp -r node_modules/font-awesome/css/font-awesome.min.css dist/css && shx cp -r node_modules/font-awesome/fonts dist",
+    "bootstrap:prefix": "postcss --use autoprefixer --autoprefixer.browsers 'last 3 versions' -o dist/css/mofo-bootstrap.css dist/css/mofo-bootstrap.css",
+    "copy:index": "shx cp index.html dist/index.html",
+    "copy:manifest": "shx cp index.html dist/index.html",
```

Comment:
this is the same command as `copy:index`, should we just remove this one and refer to `copy:index` in the task that's currently calling `copy:manifest`? (does that still need it?)